conference.js

```diff
@@ -184,29 +184,33 @@ function muteLocalVideo (muted) {
  * If requested show a thank you dialog before that.
  * If we have a close page enabled, redirect to it without
  * showing any other dialog.
- * @param {boolean} showThankYou whether we should show a thank you dialog
+ *
+ * @param {object} data Feedback data
+ * @param {boolean} data.showThankYou - whether we should show thank you dialog
+ * @param {boolean} data.feedbackSubmitted - whether feedback was submitted
  */
```

Comment:
Hm I like that we list the options that we're actually using inside the function, but from a jsdoc validation tool point of view I'm not sure what's the policy there.